portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,10 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+// Add fetch data
```

Comment:
This comment needs a bit more detail to help the reader understand. What is the fetch data? Where are we adding it to?

Reply:
Sounds good, will change!